bin/cmd.js

```diff
@@ -828,20 +695,20 @@ function drawTorrent (torrent) {
           : `${Math.floor(100 * bits / piececount)}%`
       }
 
-      let str = '{3:%s} {25+magenta:%s} {10:%s} {12+cyan:%s/s} {12+red:%s/s}'
+      let str = chalk`%s {magenta %s} %s {cyan %s} {red %s}`
 
       const args = [
-        progress,
-        wire.remoteAddress
+        progress.padEnd(3),
+        (wire.remoteAddress
           ? `${wire.remoteAddress}:${wire.remotePort}`
-          : 'Unknown',
-        prettierBytes(wire.downloaded),
-        prettierBytes(wire.downloadSpeed()),
-        prettierBytes(wire.uploadSpeed())
+          : 'Unknown').padEnd(25),
+        prettierBytes(wire.downloaded).padEnd(10),
+        (prettierBytes(wire.downloadSpeed()) + '/s').padEnd(12),
+        (prettierBytes(wire.uploadSpeed()) + '/s').padEnd(12)
       ]
 
       if (argv.verbose) {
-        str += ' {15+grey:%s} {10+grey:%s}'
+        str += chalk` {grey %s} {grey %s}`
```

Comment:
I know this syntax requires a min node version, which is it?

Reply:
chalk requires >= 12.17 (LTS). 
If u meant tagged template literals, they are supported in all nodejs versions I believe (check ECMAscript support matrix).

Reply:
awesome!